src/ebay-fake-menu-button/menu-button.tsx

```diff
@@ -65,36 +84,50 @@ const EbayMenuButton: FC<EbayButtonProps & MenuButtonProps> = ({
         })
     }
 
-    return (
-        <span {...rest} className={classnames('fake-menu-button', className)}>
-            <EbayButton
-                ref={ref as any}
-                aria-controls={menuId}
-                aria-expanded={!!expanded}
-                aria-haspopup
-                aria-label={a11yText}
-                bodyState="expand"
-                borderless={borderless}
-                className="fake-menu-button__button"
-                onClick={() => setExpanded(!expanded)}
-            >
-                {label || text || null}
-            </EbayButton>
+    const menuClasses = classnames('fake-menu-button__menu', {
+        'menu-button__menu--fix-width': fixWidth,
+        'menu-button__menu--reverse': reverse
+    })
+
+    const buttonProps: Omit<ButtonProps, 'type' | 'ref'> = {
+        ref: ref as any,
```

Comment:
I think we can remove `as any` from `ref as any` but also fine to leave it as-is.

Reply:
didn't work without `any` for me